ocs/defaults.py

```diff
@@ -18,7 +18,10 @@
 
 THIS_DIR = os.path.dirname(os.path.abspath(__file__))
 TOP_DIR = os.path.dirname(THIS_DIR)
-TEMPLATE_DIR = os.path.join(TOP_DIR, "templates/ocs-deployment/")
+TEMPLATE_DIR = os.path.join(TOP_DIR, "templates/")
+OCS_DEPLOYMENT_TEMPLATE_DIR = os.path.join(TEMPLATE_DIR, "ocs-deployment/")
```

Comment:
Name clash with PR #164

Reply:
In case https://github.com/red-hat-storage/ocs-ci/pull/164 will get merged before this one I'll adjust it here and vice versa